From 0915f90c86ce7eebd146f550ccaad14306728515 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Thu, 10 Jul 2014 01:04:04 -0400
Subject: [pci] acpi: Use dev_dbg() instead of dev_info() in acpi_pci_set_power_state()

Message-id: <20140710010403.22146.68708.stgit@gir.stowe>
Patchwork-id: 85595
O-Subject: [RHEL7.1 PATCH 056/105] PCI / ACPI: Use dev_dbg() instead of dev_info() in acpi_pci_set_power_state(
Bugzilla: 1114228
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1114228
Upstream Status: d010e5769a5ab2ae8d2bcb36e77b98172c24d80c

commit d010e5769a5ab2ae8d2bcb36e77b98172c24d80c
Author: Lan Tianyu <tianyu.lan@intel.com>
Date:   Tue Jul 30 10:32:30 2013 +0800

    PCI / ACPI: Use dev_dbg() instead of dev_info() in acpi_pci_set_power_state(

    acpi_pci_set_power_state() uses dev_info() to print diagnostic
    messages regarding ACPI power state changes of devices, but that
    results in too much not really interesting output into the kernel
    log in some cases.

    For this reason, change it to use dev_dbg() instead and prevent
    kernel log from being spammed.

    [rjw: Changelog]
    References: https://bugzilla.kernel.org/show_bug.cgi?id=60636
    Suggested-by: Alan Stern <stern@rowland.harvard.edu>
    Signed-off-by: Lan Tianyu <tianyu.lan@intel.com>
    Acked-by: Bjorn Helgaas <bhelgaas@google.com>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/pci/pci-acpi.c b/drivers/pci/pci-acpi.c
index ee9ea8e..b2d8cba 100644
--- a/drivers/pci/pci-acpi.c
+++ b/drivers/pci/pci-acpi.c
@@ -210,7 +210,7 @@ static int acpi_pci_set_power_state(struct pci_dev *dev, pci_power_t state)
  }
 
  if (!error)
-  dev_info(&dev->dev, "power state changed by ACPI to %s\n",
+  dev_dbg(&dev->dev, "power state changed by ACPI to %s\n",
     acpi_power_state_string(state_conv[state]));
 
  return error;
-- 
1.7.1